9.5/root/usr/share/container-scripts/postgresql/README.md

```diff
@@ -235,6 +235,45 @@ enough space for the copy;  upgrade failure because of not enough space might
 lead to data loss.
 
 
+Extending image
+----------------
+
+This image can be extended using
+[source-to-image](https://github.com/openshift/source-to-image).
+
+For example to build customized image `new-postgresql`
+with configuration in `~/image-configuration/` run:
+
+
+```
+$ s2i build ~/image-configuration/ postgresql new-postgresql
+```
+
+The directory passed to `s2i build` should contain one or more of the
+following directories:
+
+##### `postgresql-config/`
+
+contained files will be included at the end of image postgresql.conf file during database initialization
```

Comment:
Only `*.conf` files will be included. Slightly clearer would be `postgresql-init-config` (since this is only used during initialization).

Reply:
But, OTOH, why not to drop this hook at all, until it is explicitly requested?  It is pretty brittle, and easily implementable through `init/` by simply appending to the config file in the shell file.  Note that we are not even testing that the include statement actually works...

Reply:
Comments are still valid, but if not dropping this (smaller number of hooks, the less work with maintenance) - at least please document that *.conf files are included.